app/assets/javascripts/postScribe.js

```diff
@@ -0,0 +1,23 @@
+var waitingOnPostscribe = setInterval(function () {
```

Comment:
I would put this in `app/assets/javascripts/utilities` for lack of a better place :D

Reply:
what about a ` deferred` folder, so we know exactly what's the purpose in the path?

Reply:
Mmm I'm not sure, `defer` looks like a propery of `script`, not a name in the domain. It's something we apply to the script externally.

Reply:
ok go for utilities :)